tiled/server/schemas.py

```diff
@@ -294,6 +294,7 @@ class DeviceCode(pydantic.BaseModel):
 class PrincipalType(str, enum.Enum):
     user = "user"
     service = "service"
+    jwt_token = "jwt_token"
```

Comment:
Shouldn't the jwt_token identify the `user` or `service` to which it belongs? In the `sub` of the access token or the identity token? We may need some predicate that determines if a particular subject (`k8s-i22-beamline` | `qwe67581`) is a `user` or a `service` - there may be insight in this thread? We might need to check with LDAP.... https://diamondlightsource.slack.com/archives/C03P6QB9589/p1758275763418459